assets/css/src/components-plugin/_segments.scss

```diff
@@ -5,7 +5,8 @@
   grid-column: span 2;
 }
 
-.mailpoet-segments-segments-section .mailpoet-grid-three-columns {
+.mailpoet-segments-grid {
+  grid-template-columns: 200px 1fr 1fr;
```

Comment:
I noticed that this rule overrides the rule for mobile view defined for `mailpoet-grid-three-columns` and it results in worse UX on mobiles. There probably won't be many users who configure segments on mobile, but I guess it should be an easy fix.
<img width="1241" alt="Screenshot 2021-12-29 at 11 04 55" src="https://user-images.githubusercontent.com/1082140/147651084-ae08e706-3a68-4aad-9e80-3a6043082f1d.png">

Reply:
Good catch! Fixed